cmd/git-sync/webhook.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// WebHook structure
+type Webhook struct {
+	// URL for the http/s request
+	URL string
+	// Method for the http/s request
+	Method string
+	// Code to look for when determining if the request was successful.
+	//   If this is not specified, request is sent and forgotten about.
```

Comment:
How would this ever be `nil`  if the flag has a default?  Should default be fire-and-forget?